runtime/vm/jvminit.c

```diff
@@ -2057,7 +2057,14 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 			 * defer initialization of network data until after heap allocated, since the intialization
 			 * can initiate DLL loads which prevent allocation of large heaps.
 			 */
-			populateRASNetData(vm, vm->j9ras);
+			argIndex = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXREADIPINFOFORRAS, NULL);
+			argIndex2 = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXNOREADIPINFOFORRAS, NULL);
+			if (argIndex >= argIndex2) {
+				populateRASNetData(vm, vm->j9ras);
+				JVMINIT_VERBOSE_INIT_VM_TRACE(vm, "\nResolver for hostname used\n");
```

Comment:
Similarly to the option name, I think we should change this message.
"Using network query to determine host name and IP Address for RAS."
Plus it should be printed before populateRASNetData() is called, so if there is a delay when doing the network query, the message will be seen before the delay occurs. 

Please run with -verbose:init and take a look at how this message fits with the surrounding text. It may need to be indented.